<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPanel</class>
 <widget class="QDialog" name="UserPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Javanese Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aviable books</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtInfo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFind">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Find!</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listAviableBooks">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>501</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="listMyBooks">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>130</y>
     <width>501</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Javanese Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your books</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMyAllBooks">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAllBooks">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Show all!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
